src/Jackett/Security.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
```

Comment:
Can you not include the classes unrelated to HDTorrents in this pull request? Not to be a pain the ass - just gotta make sure we can review the feature thoroughly once you've completed it.